packages/backend-next/src/index.ts

```diff
@@ -60,4 +61,6 @@ backend.add(kubernetesPlugin());
 backend.add(permissionPlugin());
 backend.add(permissionModuleAllowAllPolicy());
 
+backend.add(catalogModuleUnprocessedEntities());
```

Comment:
broken at the moment, should be fixed by https://github.com/backstage/backstage/pull/17953

Reply:
Rebased and verified as working! :tada: